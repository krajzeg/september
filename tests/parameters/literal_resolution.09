literal := |?identifier| {
	return identifier.resolve(scope: LiteralScope)
}

print("This should just say 'identifier':", literal(identifier))
print("This should also say 'identifier':", literal("identifier"))
