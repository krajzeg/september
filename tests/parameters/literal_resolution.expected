This should just say 'identifier': identifier
This should also say 'identifier': identifier
